### ISSUE TITLE
Stress Tests

### ISSUE THREAD

**Issue by chemistry-sourabh** [CCI-MOC/ims#77]
There are some issues in BMI which do not happen regularly like target deletion failing in TGT. It is a good idea to add stress tests that will expose such issues.

**Comment by apoorvemohan** [comment]
I agree. 
@sirushtim @ravisantoshgudimetla Any suggestions on how to proceed on this?

**Comment by ravisantoshgudimetla** [comment]
I think, this is less priority as of now but I guess, its better to have one more tests folder apart from unit and integration which has stress tests alone. 

Example: For tgt, as a start create 100 targets and delete them. They could be done in parallel as well.

**Comment by sirushtim** [comment]
Yeah, I agree, once we're done with unit and functional tests, we can then move on to add stress tests to handle edge cases.

An idea to test for intermittent issues would be to execute parallel bmi pro/dpro threads in a loop and randomly write rules that will block API calls made by BMI to a service, for example, block traffic to Ceph API via IPTables, kill the tgt service, etc. This will test BMI's resilience. So I agree with @ravig, I don't think we're at that stage yet so this isn't p1 priority IMO.
